dom.bs

```diff
@@ -2645,12 +2645,69 @@ of a <var>node</var> into a <var>parent</var> before a <var>child</var>, run the
 
 <p><a lt="Other applicable specifications">Specifications</a> may define
 <dfn export id=concept-node-insert-ext>insertion steps</dfn> for all or some <a for=/>nodes</a>. The
-algorithm is passed <var ignore>insertedNode</var>, as indicated in the <a for=/>insert</a>
-algorithm below.
+algorithm is passed <var>insertedNode</var>, as indicated in the <a for=/>insert</a> algorithm
```

Comment:
Would be good to add a bit of a motivation for the separation into two sets of steps

Reply:
Hmm you mean describe the motivation for having both `#concept-node-insert` and `#concept-node-insert-ext`, or having both `#concept-node-insert-ext` and `#concept-node-post-insert-ext`? Assuming you mean the latter, do you mean just clarifying that we don't want script to run in between individual node insertions, because it can modify the DOM tree and complicate the ongoing insertion?

Reply:
Yes, the latter.  A motivation for the whole thing of running some of the insertion steps after completing the whole mutation

Reply:
I've elaborated on this a bit in the post-insertion steps note. Let me know if you had something else in mind.